### PR TITLE
HOMS-111 Add seeds.rb

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -38,7 +62,7 @@ def change
     adjustResult:
       <<: *common
       label: Adjust Result",
-"common: &common
+      "common: &common
```
Comment (sugarfree1) [inline db/seeds.rb]: Return previous indent

**Comment by GideonEXE** [comment]
UPD

File: db/seeds.rb [hunk 2]
```diff
@@ -1,6 +1,30 @@
-class AddInitialOrderTypes < ActiveRecord::Migration
-  def change
-    [
+def seed
+  add_admin()
```
Comment (sugarfree1) [inline db/seeds.rb]: Brackets are not needed.

**Comment by GideonEXE** [comment]
UPD

**Pull request merged**